torchvision/transforms/functional.py

```diff
@@ -223,9 +223,11 @@ def to_pil_image(pic, mode=None):
             pic = np.expand_dims(pic, 2)
 
     npimg = pic
-    if isinstance(pic, torch.FloatTensor) and mode != 'F':
+    if pic.is_floating_point() and mode != 'F':
         pic = pic.mul(255).byte()
     if isinstance(pic, torch.Tensor):
+        if pic.device != torch.device("cpu"):
+            pic = pic.cpu()
```

Comment:
You can call `.cpu()` unconditionally, as it doesn't do an extra copy if the tensor is already on the CPU
```suggestion
        pic = pic.cpu()
```